### PR TITLE
fixed ordering of dropout layers

### PR DESCRIPTION
**Pull request by hstojic**
dropout should be applied after the activation function, on the output - in the original code there was a dropout on inputs as well, simple correction - seems to eliminate weird training loss spikes that could be seen in integration test test_predictions.py

### CODE REVIEW

**Review by willcowley — APPROVED** [review]
LGTM - good spot!
